x-pack/plugins/watcher/public/application/sections/watch_edit/components/threshold_watch_edit/action_fields/webhook_action_fields.tsx

```diff
@@ -254,6 +254,10 @@ export const WebhookActionFields: React.FunctionComponent<Props> = ({
               defaultMessage: 'Code editor',
             }
           )}
+          setOptions={{
```

Comment:
The typings provided by the newer version of `react-ace` require us to provide this prop.

Reply:
nit: Could we provide a default value inside `EuiCodeEditor` so that the consumer don't need to?

Reply:
👍 Good idea!